### DISCUSSION TITLE
How to debug in maizzle?

### DISCUSSION THREAD

**Discussion by shivakantshukla55** [maizzle/framework#512] [Q&A]
I am using a complex object which i want to iterate based and based on some conditions i want to display it. How can i check what key or value i am getting?

**Comment by cossssmin** [comment]
Where do you need to do the check, in the Maizzle config or in the template?

In the config, you could write your checks inside an [Event](https://maizzle.com/docs/events/), and in the template you can use [conditionals](https://maizzle.com/docs/tags/#conditionals).
